static/js/components/UninitializedDBMessage.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+
+const UninitializedDBMessage = () => (
+  <div>
+    Welcome to SkyPortal. The Sources table is currently empty. <br />
+    For help with initializing the database, see the&nbsp;
+    <a href="https://skyportal.io/docs/setup.html">getting started documentation</a>.
```

Comment:
Great, and let's double check that the documentation shows how to ingest data.  (`make data`)